scripts/drop_index.js

```diff
@@ -17,10 +17,12 @@ function drop(){
   });
 }
 
+// check all hosts to see if any is not localhost
 function warnIfNotLocal() {
-  if(config.esclient.hosts[0].host !== "localhost") {
+  if (config.esclient.hosts.some((env) => { return env.host !== 'localhost'; } )) {
     console.log(colors.red("WARNING: DROPPING SCHEMA NOT ON LOCALHOST"));
```

Comment:
But wouldn't having multiple hosts listed cause the index to be dropped in all those hosts?